fwoa-tools/src/binaryConverter.ts

```diff
@@ -65,10 +39,18 @@ async function getBinaryResource(itemKey: string): Promise<GetObjectOutput> {
   return file;
 }
 
-async function getBinaryObject(itemKey: string, versionId: number = 1): Promise<GetObjectOutput> {
+export async function getBinaryObject(
+  itemKey: string,
+  versionId: number = 1,
+  s3Client: S3
+): Promise<GetObjectOutput> {
+  let tenantPrefix = '';
+  if (process.env.MIGRATION_TENANT_ID) {
+    tenantPrefix = `${process.env.MIGRATION_TENANT_ID}/`;
+  }
   const files = await s3Client
     .listObjectsV2({
-      Bucket: BINARY_BUCKET_NAME!,
+      Bucket: process.env.BINARY_BUCKET_NAME!,
```

Comment:
The reason I had to switch to using process.env directly is so that the updates to process.env in the unit tests is actually processed. If there's a better way to do this I can switch to that